app/components/course-page/course-stage-step/second-stage-instructions-card/implement-solution-step.ts

```diff
@@ -9,10 +9,15 @@ interface Signature {
     repository: RepositoryModel;
     courseStage: CourseStageModel;
     isComplete: boolean;
+    isShowingLanguageGuide: boolean;
   };
 }
 
-export default class ImplementSolutionStepComponent extends Component<Signature> {}
+export default class ImplementSolutionStepComponent extends Component<Signature> {
+  get currentLanguageName(): string {
+    return this.args.repository.language!.name;
```

Comment:
@andy1li this level of indirection usually isn't worth it - can use `@repository.language.name` directly in the template 🙂 Also, I think that this card _might_ render when a language isn't selected, so in that case using the template method won't throw an exception (templates tolerate null values and just render empty contents, vs. this method would actually through an exception)